ReSharper.FSharp/src/FSharp.Common/src/Checker/FSharpCheckerService.fs

```diff
@@ -98,8 +98,11 @@ type FSharpCheckerService
         let source = FSharpCheckerService.getSourceText file.Document
         logger.Trace("ParseAndCheckFile: start {0}, {1}", path, opName)
 
+        let reactorMonitor = file.GetSolution().GetComponent<IFcsReactorMonitor>()
```

Comment:
Although it'd be not very good or pretty but we could pass the reactor monitor the same way it's done in `FcsProjectProvider`, so it'd always be available inside the checker service.